### PR TITLE
Feat/node: Change bigint to u64 and u256 alias

### PR DESCRIPTION
**Pull request by kwek20**
Addressing the comment from here: https://github.com/iotaledger/iota-sdk/pull/1034#discussion_r1297275562

### CODE REVIEW

File: bindings/nodejs/lib/types/block/output/token-scheme.ts [hunk 1]
```diff
@@ -26,15 +27,11 @@ abstract class TokenScheme {
  * Simple token scheme.
  */
 class SimpleTokenScheme extends TokenScheme {
-    private mintedTokens: bigint;
-    private meltedTokens: bigint;
-    private maximumSupply: bigint;
+    private mintedTokens: u64;
+    private meltedTokens: u64;
+    private maximumSupply: u64;
```
Comment (Thoralf-M) [inline bindings/nodejs/lib/types/block/output/token-scheme.ts]: These are U256

Comment (Thoralf-M) [reply bindings/nodejs/lib/types/block/output/token-scheme.ts]: Also the other native token things below

Comment (kwek20) [reply bindings/nodejs/lib/types/block/output/token-scheme.ts]: shit didnt even think of that madness

File: bindings/nodejs/lib/types/utils/type_aliases.ts [hunk 2]
```diff
@@ -3,3 +3,6 @@
 
 export type u16 = number;
 export type u64 = bigint;
+
+// U256 mainly used for token amount
```
Comment (kwek20) [inline bindings/nodejs/lib/types/utils/type_aliases.ts]: do we care about this comment?

File: bindings/nodejs/lib/types/block/output/token-scheme.ts (hunk 1 above)
Comment (kwek20) [reply bindings/nodejs/lib/types/block/output/token-scheme.ts]: fixed

**Review by Thoralf-M — DISMISSED** [review]

File: bindings/nodejs/lib/types/utils/type_aliases.ts (hunk 2 above)
Comment (thibault-martinez) [reply bindings/nodejs/lib/types/utils/type_aliases.ts]: I'd personally remove it, not very useful

File: bindings/nodejs/lib/types/client/output_builder_params/basic-output-params.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Copyright 2021-2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
-import { UnlockCondition, Feature, INativeToken } from '../..';
+import { UnlockCondition, Feature, INativeToken, u64 } from '../..';
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/client/output_builder_params/basic-output-params.ts:4]:
```suggestion

import { UnlockCondition, Feature, INativeToken, u64 } from '../..';
```

File: bindings/nodejs/lib/types/models/native-token.ts [hunk 4]
```diff
@@ -1,6 +1,7 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
+import { u256 } from '../utils';
 import type { HexEncodedString } from '../utils/hex-encoding';
 /**
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/types/models/native-token.ts:7]:
```suggestion

/**
```

**Review by DaughterOfMars — DISMISSED** [review]

**Comment by thibault-martinez** [comment]
I guess this will conflict when we merge https://github.com/iotaledger/iota-sdk/pull/1034

**Review by Thoralf-M — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Review by qrayven — APPROVED** [review]

**Pull request merged**